spec/requests/create_new_conversion_project_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe "Create new conversion project" do
```

Comment:
I think this could be a `conversion_project_controller_spec`, to match some of the others, but let me know what you think!

Reply:
I've always done these as a `controller` spec, but I might be wrong to do that. [Theres definitely a few people who do that but not loads](https://github.com/search?p=1&q=org%3ADFE-Digital+filename%3Acontroller+path%3Aspec%2Frequests+language%3ARuby+language%3ARuby&type=Code) 

I think from my point of view it's easy to ensure you have full coverage by testing the controller actions and just having the naming match. No extra thought required 🤷 "Where shall I put the test for the new controller action I have just written in the notes_controller" -> "notes_controller_spec"

Looks like we aren't meant to use controller spec regardless 
<img width="748" alt="image" src="https://user-images.githubusercontent.com/47089130/204317601-ec8b0a23-8c1a-4a06-8588-fa7469b5f9fe.png">